deltacat/utils/ray_utils/retry_handler/task_constants.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
We may need very different constants altogether. This could stay out of skeleton CR.